Classes/Public/Manager/OCTManagerConstants.h

```diff
@@ -53,3 +53,20 @@ typedef NS_ENUM(NSUInteger, OCTMessageFileType) {
      */
     OCTMessageFileTypeReady,
 };
+
+typedef NS_ENUM(NSInteger, OCTMessageCallType) {
```

Comment:
If you will check `OCTDBMessageFile` you will see that `fileType` is actually an int. `OCTMessageFileType` is just implicitly casted to it